### ISSUE TITLE
cell adhesion molecule

### ISSUE THREAD

**Issue by RLovering** [geneontology/go-ontology#14533]
Hi Pascale

I have opened a ticket to discuss the issue you raised in https://github.com/geneontology/go-ontology/issues/14324 where you express concern about the term: 'GO:0086080 | protein binding involved in heterotypic cell-cell adhesion'
this may lead to an explosion of terms if we allow this pattern (unless there is something specific about this term that isn't immediately obvious). In this case would the MF term GO:0098631 'cell-cell adhesion mediator activity' represent the function you are trying to capture ?

The main reason this term was created was because the cell adhesion molecule activity term was obsoleted and this was a way of creating the term required to capture this data. The  'cell-cell adhesion mediator activity' term meets this need. However, it is possible that some of the annotations are no associated with cell adhesion molecules. So these will need to be checked before removing this term. 

As this term has 5 cardiac child terms can we delay the removal of this term and the revision of the annotations until after news about the cardiac paper please?

please add GO:0098631 'cell-cell adhesion mediator activity' in the list of replace by for 
GO:0005194 obsolete cell adhesion molecule activity

Thanks

Ruth

**Comment by ValWood** [comment]
I notice we have used
"cell adhesion molecule binding" to represent this (presumably because any adhesion molecule must bind to another adhesion molecule!)
http://pombase-dev.bioinformatics.nz/term/GO:0050839

I didn't know about 
GO:0098631 'cell-cell adhesion mediator activity

Maybe  GO:0050839 is not really required  either if this is how it has been used?

**Comment by pgaudet** [comment]
> please  add GO:0098631 'cell-cell adhesion mediator activity' in the list of replace by for
GO:0005194 obsolete cell adhesion molecule activity

DONE

**Comment by pgaudet** [comment]
Hi @RLovering 
Paul and I renamed '**GO:0098631 protein binding involved in cell adhesion**' -> 'cell adhesion mediator activity' in September when we started the MF refactor. 

We also changed the definition from 'Any protein binding that is involved in cell adhesion.' to 'Any binding by a cell surface protein to a molecule external to the cell, that meditates adhesion of the cell to that external substrate.' **I am just now noticing that the former parent (protein binding) is therefore be incorrect; I now to moved it up to 'binding'. Same for cell-matrix adhesion mediator activity. (Neither term have any direct annotations).** 

I suspect there are a large number of different combinations of cells that can bind to each other; isn't it possible to capture that information as an extension, which would then cover 'protein binding involved in heterotypic cell-cell adhesion'? 

@ValWood
I agree that 'cell adhesion molecule binding' is not very informative regarding the protein function: it may be binding a ligand, or a regulator, or a member of the complex... but we have many types of x-binding, for now I prefer not to open that.
